### PR TITLE
Use .env vars in build, update webpack, remove hardcoded cms-prototype-blocks

### PR DESCRIPTION
**Pull request by sy238-unikent**
Makes the content of the .env file available in the webpack build script as process.ENV.{variable-name}

Replaces the hard-coded imports from cms-prototype-blocks in Block.vue and Preview.vue with an alias "theme" which is aliased to whatever is set for DEFINITIONS_PATH in the .env file.

This allows defintion repositories to be named something other than "cms-prototype-blocks" and means they no longer HAVE to be installed in node_modules as a dependency.

This should also allow us to specify other build-time variables, for example URLs, paths, etc, and thus
aid in the separation of editor from API.

As my knowledge of webpack is limited, there is the possibility that this also breaks things in ways I cannot
begin to comprehend...

### CODE REVIEW

**Comment by theninja** [comment]
I'm keen to get this merged when I get back to the office as it's a real step in the right direction.
I'll do a bit of digging to make sure this doesn't have any unintended side-effects.

**Comment by sy238-unikent** [comment]
@theninja that would be good.

Looks like we are extending the sprint into next week too. Hopefully going to think about / implement "themes" as well, as template stuff is going nicely... might also be a good time to look at upgrading to 5.5, merging changes in your laravel fork into astro and decoupling UoK dependencies from the opensourcy bits...

**Comment by theninja** [comment]
Sounds good. I'll prepare my admin UI changes too. Seems like this could be quite a productive sprint!

File: webpack.config.babel.js [hunk 1]
```diff
@@ -33,7 +35,15 @@ export default {
 				loader: 'vue-loader',
 				options: {
 					loaders: {
-						js: 'babel-loader',
+						js: {
```
Comment (theninja) [inline webpack.config.babel.js:38]: This is one of the bits I'm not so sure about,

Comment (sy238-unikent) [reply webpack.config.babel.js:38]: IIRC I had to add this to get it to work... but now not sure why or exactly what the problems were.

Comment (sy238-unikent) [reply webpack.config.babel.js:38]: I was getting lots of errors when referencing a definitions folder stored outside of node_modules.

**Pull request merged**